config/initializers/prosody.rb

```diff
@@ -0,0 +1,3 @@
+Prosody.config({
+  file: "config/prosody.cfg.lua"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.